### ISSUE TITLE
Update table structure in docs

### ISSUE THREAD

**Issue by iuvbio** [freqtrade/freqtrade#1629]
The [table structure](https://www.freqtrade.io/en/latest/sql_cheatsheet/) in the docs is not complete.

What's in the docs:
```
CREATE TABLE trades (
    id INTEGER NOT NULL,
    exchange VARCHAR NOT NULL,
    pair VARCHAR NOT NULL,
    is_open BOOLEAN NOT NULL,
    fee_open FLOAT NOT NULL,
    fee_close FLOAT NOT NULL,
    open_rate FLOAT,
    open_rate_requested FLOAT,
    close_rate FLOAT,
    close_rate_requested FLOAT,
    close_profit FLOAT,
    stake_amount FLOAT NOT NULL,
    amount FLOAT,
    open_date DATETIME NOT NULL,
    close_date DATETIME,
    open_order_id VARCHAR,
    PRIMARY KEY (id),
    CHECK (is_open IN (0, 1))
);
```

What the table structure really is:
```
CREATE TABLE trades (
	id INTEGER NOT NULL, 
	exchange VARCHAR NOT NULL, 
	pair VARCHAR NOT NULL, 
	is_open BOOLEAN NOT NULL, 
	fee_open FLOAT NOT NULL, 
	fee_close FLOAT NOT NULL, 
	open_rate FLOAT, 
	open_rate_requested FLOAT, 
	close_rate FLOAT, 
	close_rate_requested FLOAT, 
	close_profit FLOAT, 
	stake_amount FLOAT NOT NULL, 
	amount FLOAT, 
	open_date DATETIME NOT NULL, 
	close_date DATETIME, 
	open_order_id VARCHAR, 
	stop_loss FLOAT, 
	initial_stop_loss FLOAT, 
	stoploss_order_id VARCHAR, 
	stoploss_last_update DATETIME, 
	max_rate FLOAT, 
	sell_reason VARCHAR, 
	strategy VARCHAR, 
	ticker_interval INTEGER, 
	PRIMARY KEY (id), 
	CHECK (is_open IN (0, 1))
);
```

Unless this is intentional to simplify, in which case it should probably be mentioned.

**Comment by xmatthias** [comment]
No intend - just oversight when we were adding new columns to the table.

**Comment by xmatthias** [comment]
Closed in #1632 :+1: 
Thanks @iuvbio
